workflow/travis.md

```diff
@@ -1 +1,31 @@
 # Integrating with Travis CI
+
+To integrate npm Enterprise with Travis CI, you need to generate a `.npmrc`
+that knows about your npm Enterprise instance and its credentials:
+
+1. Fetch your npm Enterprise secret token:
+
+  Look in your `~/.npmrc` file, you will see an entry that looks something like this:
```

Comment:
Should we perhaps mention the ability to generate tokens specifically for CI via `npme manage-tokens generate`?